Proiect-PAO/src/Main.java

```diff
@@ -1,45 +1,112 @@
-                case "get_locations_by_desc_capacity" -> service.listLocationsByDescCapacity();
-                case "get_events" -> service.listEvents();
-
-                case "get_tickets_of_clientID" -> service.listClientTickets(in);
-
-                case "get_events_from" -> service.listAllEventsStartingFrom(in);
-                case "get_events_until" -> service.listAllEventsUntil(in);
-
-                case "sell_full" -> service.sellAdultFullPassForCategory(in);
-                case "sell_day" -> service.sellAdultDayPassForCategory(in);
-                case "sell_day_discount" -> service.sellDiscountedDayPassForCategory(in);
+                case "add_client" -> {
+                    service.createClient(in);
+                    audit.logAction("add_client");
+                }
+                case "add_location" -> {
+                    service.addLocation(in);
+                    audit.logAction("add_location");
+                }
+                case "add_sectioned_loc" -> {
+                    service.addSectionedLocation(in);
+                    audit.logAction("add_sectioned_loc");
+                }
+                case "add_event" -> {
+                    service.createEvent(in);
+                    audit.logAction("add_event");
+                }
+                case "add_festival" -> {
+                    service.createFestival(in);
+                    audit.logAction( "add_festival");
+                }
+                case "get_clients" -> {
+                    List<Client> c = service.getClients();
+                    out.writeToFile("data/output.txt", c);
+                    audit.logAction("get_clients");
+                }
+                case "get_locations" -> {
+                    List<Location> l = service.getLocations();
+                    out.writeToFile("data/output.txt", l);
+                    audit.logAction("get_locations");
+                }
+                case "get_locations_by_desc_capacity" -> {
+                    service.listLocationsByDescCapacity();
+                    audit.logAction("get_locations_by_desc_capacity");
+                }
+                case "get_events" -> {
+                    List<Event> e = service.getEvents();
+                    out.writeToFile("data/output.txt", e);
+                    audit.logAction("get_events");
+                }
+                case "get_tickets" -> {
+                    List<Ticket> t = service.getTicketsSold();
+                    out.writeToFile("data/output.txt", t);
+                    audit.logAction("get_tickets");
+                }
+
+                case "get_tickets_of_clientID" -> {
+                    service.listClientTickets(in);
+                    audit.logAction("get_tickets_of_clientID");
+                }
+
+                case "get_events_from" -> {
+                    service.listAllEventsStartingFrom(in);
+                    audit.logAction("get_events_from");
```

Comment:
what happens if listAllEventsStartingFrom receives an invalid data from user input? then, we receive an exception, and line 92 is not executed, program is stopped, even if the method was actually called, the audit service will not write this action in the audit.csv. Maybe consider treating this exception so that the program recovers by displaying a message to the user (Please insert a date in the following format: ...) and the action (even if was not successfull) will be logged in audit.csv in order to keep track